core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/views/YouTubePlayerView.kt

```diff
@@ -200,4 +226,16 @@ class YouTubePlayerView(context: Context, attrs: AttributeSet? = null, defStyleA
 
     fun removeFullScreenListener(fullScreenListener: YouTubePlayerFullScreenListener): Boolean =
             fullScreenHelper.removeFullScreenListener(fullScreenListener)
+
+
+    /*function which sets OntouchEvent Listener*/
+    fun setOnTouchListener(onTouchEvent: OnTouchEvent) {
+        if (this.onTouchEvent == null) {
```

Comment:
Why the check for null?

Reply:
I case User can set multiple TouchListener that why for the sake of single reference.